gnucash/gnome/assistant-stock-transaction.cpp

```diff
@@ -633,26 +634,55 @@ add_error_str (StringVec& errors, const char* str)
     errors.emplace_back (_(str));
 }
 
+typedef struct
+{
+    bool debit_side;
+    bool value_is_zero;
+    std::string account;
+    std::string memo;
+    std::string value;
+    std::string units;
+} SummaryLineInfo;
+
 static void
-check_page (GtkListStore *list, gnc_numeric& debit, gnc_numeric& credit,
+add_to_summary_table (GtkListStore *list, SummaryLineInfo line)
+{
+    GtkTreeIter iter;
+    auto tooltip = g_markup_escape_text (line.memo.c_str(), -1);
+    gtk_list_store_append (list, &iter);
+    gtk_list_store_set (list, &iter,
+                        SPLIT_COL_ACCOUNT, line.account.c_str(),
+                        SPLIT_COL_MEMO, line.memo.c_str(),
+                        SPLIT_COL_TOOLTIP, tooltip,
+                        SPLIT_COL_DEBIT, line.debit_side ? line.value.c_str() : "",
+                        SPLIT_COL_CREDIT, !line.debit_side ? line.value.c_str() : "",
+                        SPLIT_COL_UNITS, line.units.c_str(),
+                        -1);
+    g_free (tooltip);
+}
+
+
+static void
+check_page (SummaryLineInfo& line, gnc_numeric& debit, gnc_numeric& credit,
             FieldMask splitfield, Account *acct, GtkWidget *memo, GtkWidget *gae,
             gnc_commodity *comm, const char* page, StringVec& errors)
 {
-    if (splitfield == FieldMask::DISABLED)
-        return;
     const char* missing_str = N_("(missing)");
```

Comment:
This needs a translator comment explaining what might be missing and maybe that we hope there aren't any languages with a different word for missing accounts and missing amounts.